presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetDataSource.java

```diff
@@ -19,6 +19,8 @@
 public interface ParquetDataSource
         extends Closeable
 {
+    long getReadBytes();
```

Comment:
Should we call these two methods `getUncompressedSize()` and `getCompressedSize()` for clarity?